src/i18n/locales/en/translation.json

```diff
@@ -358,6 +361,30 @@
         }
       }
     },
+    "choose_umbrel_connector": {
```

Comment:
First I would need to translate a title by the path (e.g. "Umbrel"), then send it as a parameter to the translation for the whole sentence (e.g. "Connect to {title}"). To be able to do this I have to create a function and then overwrite the title like

```
function getDistribution() {
    const name = t()
    return {
        ...connectorMap["umbrel"],
        title: <translate title sentence and pass translated name as param>
    }
```

This would be fine, the problem is I cannot overwrite the `<ChooseConnector>` props

```
  umbrel: {
    path: "umbrel",
    element: (
      <ChooseConnector
        title={i18n.t("translation:choose_umbrel_connector.title")}
        description={i18n.t("translation:choose_umbrel_connector.description")}
        connectorRoutes={umbrelConnectorRoutes}
      />
    ),
    title: i18n.t("translation:choose_connector.umbrel.title"),
    logo: umbrel,
    children: umbrelConnectorRoutes,
  },
```

Reply:
@reneaaron What do you think? Is it ok for you?

Reply:
@lujakob I think I'll have to wrap my head around that. Particularly not happy about introducing all of those new translations that are literally the same. (except for the connector name)

Reply:
@reneaaron I have an idea now, I think that might work.

Reply:
@reneaaron I could improve it by adding a function. Please have a look and let me know what you think. Unfortunately I could not fix the type error for dynamic `i18n` keys and added a `ts-ignore` for now. Maybe you have an idea or I stumble upon something in the meantime.

Also I had to add some interfaces and unions, because the kollider route has a different structure and approach.

Another thing that I noticed, the routing config for the welcome page didn't work. I already noticed this when I started working on this issue but forgot about it. The routing config for Options and Welcome is done differently and expect a slightly different structure for nested routes. One of them had the index page `ChooseConnector` within the children's array and `index` set, the other not.

To avoid further complicating the structure of provided routes of `getConnectorRoutes` I modified the routing strategy in welcome and adapted it to the way it is in options. Let me know what you think.